caf-audit-service-container/test-configs/cfg_test_auditservice_ElasticAuditConfiguration

```diff
@@ -0,0 +1,6 @@
+{
+    "hostAndPort": "192.168.56.10:9300",
```

Comment:
As mentions this will need to change to hostAndPortValues, this is a note to you and I not to forget this :-)